ui/pages/confirmations/components/confirm/title/title.tsx

```diff
@@ -23,15 +23,29 @@ import { useIsNFT } from '../info/approve/hooks/use-is-nft';
 import { useTokenTransactionData } from '../info/hooks/useTokenTransactionData';
 import { getIsRevokeSetApprovalForAll } from '../info/utils';
 import { getIsRevokeDAIPermit } from '../utils';
+import { useSignatureEventFragment } from '../../../hooks/useSignatureEventFragment';
+import { useTransactionEventFragment } from '../../../hooks/useTransactionEventFragment';
 import { useCurrentSpendingCap } from './hooks/useCurrentSpendingCap';
 
 function ConfirmBannerAlert({ ownerId }: { ownerId: string }) {
   const { generalAlerts } = useAlerts(ownerId);
+  const { updateSignatureEventFragment } = useSignatureEventFragment();
+  const { updateTransactionEventFragment } = useTransactionEventFragment();
 
   if (generalAlerts.length === 0) {
     return null;
   }
 
+  const onClickSupportLink = () => {
+    const properties = {
+      properties: {
+        external_link_clicked: 'security_alert_support_link',
+      },
+    };
+    updateSignatureEventFragment(properties);
+    updateTransactionEventFragment(properties, ownerId);
+  };
+
```

Comment:
Should we create this callback with `useCallback` if it has `ownerId` dependency?

```suggestion
const onClickSupportLink = useCallback(() => {
       const properties = {
         properties: {
           external_link_clicked: 'security_alert_support_link',
         },
       };
       updateSignatureEventFragment(properties);
       updateTransactionEventFragment(properties, ownerId);
     }, [ownerId]);
```

Reply:
I think only fallback is settling in the `updateTransactionEventFragment` not the given `transactionId` (in this case `ownerId`) In my opinion, it's good practice to have it right in order to prevent any side effects.

Reply:
Due to having `Rendered more hooks than during the previous render.` after using `useCallback` hook we will handle this later.